### PR TITLE
hide unused navigation tabs

### PR DESCRIPTION
**Pull request by leggsimon**
This removes the navigation bar as per the new design 

screenshot 2017-02-01 10 43 59

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .editorconfig
 .eslintrc.js
 .scss-lint.yml
+yarn.lock
```
Comment (quarterto) [inline .gitignore]: yarnpkg/yarn#1583

Comment (leggsimon) [inline .gitignore]: I know, but I'm the only one using yarn AFAIK

Comment (leggsimon) [inline .gitignore]: Having said that, I'm also the only one working on this...

Comment (quarterto) [inline .gitignore]: you could add it to `.git/config/exclude` to avoid cluttering `.gitignore`?

Comment (leggsimon) [inline .gitignore]: woah

**Pull request merged**